src/dotty/tools/dotc/transform/ElimRepeated.scala

```diff
@@ -43,6 +43,8 @@ class ElimRepeated extends MiniPhaseTransform with InfoTransformer with Annotati
           paramTypes.init :+ last
         } else paramTypes
       tp.derivedMethodType(paramNames, paramTypes1, resultType1)
+    case tp: ExprType if tp.resultType.isRepeatedParam =>
+       ExprType(tp.resultType.underlyingIfRepeated(isJava = false))
```

Comment:
I don't understand what the change achieves. elimRepeated removes repeated method parameters. It does not touch other occurrences of repeated types. In fact the expectation is that these were not generated in the first place. If elimRepeated should touch ExprTypes then it also should handle naked repeated types which are not wrapped in ExprTypes. But it does not do this.